.rubocop.yml

```diff
@@ -58,3 +58,28 @@ Sorbet/StrictSigil:
     - "lib/ruby-lsp.rb"
     - "lib/ruby_indexer/lib/ruby_indexer/prefix_tree.rb"
     - "lib/ruby_lsp/load_sorbet.rb"
+
+Layout/ClassStructure:
+  Enabled: true
+  ExpectedOrder:
+    - module_inclusion
+    - constants
+    - public_attribute_macros
+    - public_delegate
```

Comment:
`delegate` comes from rails, not necessary here

Reply:
Ah, was thinking this was Ruby's `def_delegator(s)`. Can remove.